template_config.yml

```diff
@@ -52,4 +52,7 @@ test_performance: false
 test_released_plugin_with_next_pulpcore_release: false
 test_s3: true
 update_redmine: true
+upgrade_range:
+  - pulp_ansible_branch: 0.7
+    pulpcore_branch: 3.11
```

Comment:
In the future when we are releasing pulp_ansible 0.9.0, will the upgrade tests check 0.7->0.8->0.9 and 0.7->0.9? 

Is this setting something we will change continuously from release to release or just keep constant?

Reply:
At this first moment, I see we keep it constant, but in the future, we may create some rule: "only test upgrade for latest 2 releases"

we can add entries, and test pulpcore x pulp_ansible combinations:
```yaml
upgrade_range:  
  - pulp_ansible_branch: 0.7    
    pulpcore_branch: 3.11
 - pulp_ansible_branch: 0.8    
    pulpcore_branch: 3.13
- pulp_ansible_branch: 0.8    
    pulpcore_branch: 3.14
```